MAUI/Pull-to-Refresh/customization.md

```diff
@@ -0,0 +1,640 @@
+
+{% endhighlight %}
+{% highlight c# tabtittle="MainPage.xaml.cs" %}
+
+    pullToRefresh.ProgressBackground = Color.White;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## ProgressColor
+
+The color to the progress indicator's arc. 
+
+{% tabs %}
+{% highlight xaml tabtittle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh" 
+                                ProgressColor = "Blue"/>
+
+{% endhighlight %}
+{% highlight c# tabtittle="MainPage.xaml.cs" %}
+
+    pullToRefresh.ProgressColor = Color.Blue;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## ProgressThickness
+
+The width of the progress indicator's arc. 
+
+{% tabs %}
+{% highlight xaml tabtittle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh" 
+                                ProgressThickness="5"/>
+
+{% endhighlight %}
+{% highlight c# tabtittle="MainPage.xaml.cs" %}
+
+    pullToRefresh.ProgressThickness = 5d;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## RefreshViewWidth
+
+The width of the refresh view.
+
+{% tabs %}
+{% highlight xaml tabtittle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh" 
+                                RefreshViewWidth="50"/>
+
+{% endhighlight %}
+{% highlight c# tabtittle="MainPage.xaml.cs" %}
+
+    pullToRefresh.RefreshViewWidth = 50d;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## RefreshViewHeight
+
+The height to the refresh View.
+
+{% tabs %}
+{% highlight xaml tabtittle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfPullToRefresh x:Name="pullToRefresh" 
+                                RefreshViewHeight="50"/>
+
+{% endhighlight %}
+{% highlight c# tabtittle="MainPage.xaml.cs" %}
+
+    pullToRefresh.RefreshViewHeight = 50d;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Programmatic Support 
+
+### StartRefreshing()
+
+The StartRefreshing method is used to refresh the content without interaction in pullable content. When you invoke this StartRefreshing() method,then the Progress indicator will be shown. 
+
+{% tabs %}
+{% highlight c# tabtittle="MainPage.xaml.cs" %}
+
+    pullToRefresh.StartRefreshing();
+
+{% endhighlight %}
+{% endtabs %}
+
+### EndRefreshing()
+
+The EndRefreshing method is used to end the progress animation of the `PullToRefresh`.
+
+{% tabs %}
+{% highlight c# tabtittle="MainPage.xaml.cs" %}
+
+    pullToRefresh.EndRefreshing();
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Host .NET MAUI DataGrid as pullable content
+
+The `PullToRefresh` control provides support for loading any custom control as pullable content. To host the .NET MAUI Datagrid inside the PullToRefresh, follow these steps.
+<ol>
+    <li> Add the required assembly references to the pcl and renderer projects as discussed in the Assembly deployment section of <a href="https://help.syncfusion.com/maui/datagrid/getting-started">DataGrid </a> and <a href="https://help.syncfusion.com/maui/pulltorefresh/getting-started">PullToRefresh </a>.</li>
```

Comment:
check this content and change it to MAUI standard.

Add the required assembly references as discussed in the getting-started section of ...

Reply:
changed the content